spec/views/wikies/index.html.erb_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe "wikies/index", :type => :view do
+  before(:each) do
+    assign(:wikies, [
```

Comment:
Consider using FactoryGirl's `create_list(...)` method here to reduce duplication.